docs/rules/no-extra-parens.md

```diff
@@ -20,7 +18,7 @@ This rule takes 1 or 2 arguments. The first one is a string which must be one of
 
 The second one is an object for more fine-grained configuration when the first option is "all".
 
-#### "all"
+### "all"
```

Comment:
We should try to format options the way they actually are in the config file, so strings should be in quotes, and nonstrings should be in code font.